core/src/buffer.rs

```diff
@@ -242,6 +257,167 @@ impl Slice {
     }
 }
 
+/// A list of multiple slices of the same buffer.
+///
+/// `Substream` represents a substream of the underlying buffer. Slices can be
+/// appended to the chunks list
```

Comment:
This sentence looks unfinished ... ? Anyway, it's not in the final code, so it's not important.

Reply:
I think you might have reviewed an old version of the PR - I caught and fixed this a little while ago and the fixed documentation is in my tree.

Reply:
Like I said, I'm going commit-by-commit with the review... I know it has unfortunate side effects like this, sorry.